feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -86,9 +90,26 @@ func TestCopyingDebugFiles(t *testing.T) {
 	t.Logf("Wait 60 seconds for process to restart ...")
 	time.Sleep(60 * time.Second)
 
-	componentName := map[string]string{"name": components[dut.Vendor()]}
+	componentName := map[string]string{}
```

Comment:
Please resolve static check finding:

this value of componentName is never used (SA4006)